src/modules/fancyzones/lib/util.h

```diff
@@ -96,6 +96,32 @@ namespace FancyZonesUtils
             SRCCOPY);
     }
 
+    inline COLORREF HexToRGB(const std::wstring& hex)
+    {
+        std::wstring color = hex;
+        if (hex.size() == 7)
+        {
+            color = hex.substr(1);
+        }
+        else if (hex.size() == 9)
+        {
+            color = hex.substr(3);
+        }
```

Comment:
Why is this logic in here and not directly in settings ui with a proper input validation?

Reply:
We are fixing it there as well.

Reply:
Consider that the json can be manually edit, we still want to be robust here and not make assumptions that the color format is always #ABCBEF

Reply:
Why is it then on 2 places and increasing maintainability ? If settings ui ensures correct values in settings.json, this one should not do anything

Reply:
For the reason I just explained.

Reply:
Manual editing of settings.json is not a supported way of changing settings - guarding against unsupported things is a wrong approach and as I wrote, it unnecessary decrease maintainability! Do we guard against all other changes that can be done in settings.json for every module as well?

Reply:
Manual json editing is supported. We need it to make it robust and improve the code that handles the settings.

Reply:
If it is supported as you say, than please add code that will handle following cases:
color: "blue"
color: "haha_I_can_write_here_anything_and_expect_that_it_will_work"
color: "##000000"
... and everything else...